src/Implementations/MethodDescriptor.php

```diff
@@ -32,14 +37,16 @@ class MethodDescriptor implements MDI
      * @param string $name
      * @param Type[] $params
      * @param Type $return
+     * @param boolean $static
      * @param ClassType[] $exceptions
      */
-    public function __construct($name, $params, Type $return, $exceptions = array())
+    public function __construct($name, $params, Type $return, $static = false, $exceptions = array())
```

Comment:
public function __construct($name, array $params, Type $return, $static = false, array $exceptions = array())

Reply:
but.. this should work in lower php versiosn too? then: don't care about this

Reply:
`bool $static` is not a option, should work with php5 after ally